src/fun/scala/kinesis/mock/ListStreamsTests.scala

```diff
@@ -8,7 +8,16 @@ class ListStreamsTests extends munit.CatsEffectSuite with AwsFunctionalTests {
     for {
       res <- resources.kinesisClient.listStreams().toIO
     } yield assert(
-      res.streamNames().size == 1,
+      !res.streamNames().isEmpty,
+      s"$res"
+    )
+  }
+
+  fixture.test("It should list all initialized streams") { resources =>
+    for {
+      res <- resources.kinesisClient.listStreams().toIO
+    } yield assert(
+      initializedStreams.forall { case (name,_) => res.streamNames().contains(name) },
```

Comment:
We should also do a listShards() (or describeStream()) to get the shard count and assert that.

Reply:
Also we need to test that the stream is active.

Reply:
I added these into their respective test classes (`ListShardsTests` and `DescribeStreamTests`)